server/authentication_test.go

```diff
@@ -126,13 +125,6 @@ func TestSSLEnforcement(t *testing.T) {
 		{"GET", ts.URLPrefix, nil, noCertsContext, true, http.StatusNotFound},
 		{"GET", ts.URLPrefix, nil, insecureContext, false, -1},
 
-		// /kv/db/: kv.DBServer. These are proto reqs. Node certs required.
```

Comment:
how about leaving one here here StatusNotFound. Not particularly critical though, I doubt the KV endpoint will sneak back into the code unnoticed.